build.go

```diff
@@ -193,13 +246,23 @@ func Build(f BuildFunc, options ...Option) {
 	}
 
 	var buildpackInfo struct {
-		Buildpack BuildpackInfo `toml:"buildpack"`
+		APIVersion string        `toml:"api"`
+		Buildpack  BuildpackInfo `toml:"buildpack"`
 	}
+
 	_, err = toml.DecodeFile(filepath.Join(cnbPath, "buildpack.toml"), &buildpackInfo)
 	if err != nil {
 		config.exitHandler.Error(err)
 		return
 	}
+	oldAPI := false
+	apiV05, _ := semver.NewVersion("0.5")
+	if version, err := semver.NewVersion(buildpackInfo.APIVersion); err != nil {
+		config.exitHandler.Error(err)
+		return
+	} else if version.LessThan(apiV05) {
+		oldAPI = true
+	}
```

Comment:
Rather than recording this in a variable, `oldAPI`, that will likely be superseded as soon as a newer API version comes out, can we just record the API version specified in the buildpack and then do the comparisons in the locations that `oldAPI` appears below? This would be helpful to future developers as they would more easily understand the code in that location.